cmudb/tscout/README.md

```diff
@@ -1 +1,28 @@
-# noisepage_bpf
+# Setup
+
+## Installation
+
+1. `sudo pip3 -r requirements.txt` -- the packages must be available to root's Python
```

Comment:
I think this is missing the install keyword.  Just used `sudo pip3 install -r requirements.txt` successfully